### PR TITLE
Follow up documents changes (ko)

### PR DESCRIPTION
**Pull request by riseshia**
:link: #2818

Follow up #2732, #2784, #2899, #1410, #2857, #2893

Please review this 🙏

### CODE REVIEW

File: ko/security/index.md [hunk 1]
```diff
@@ -9,21 +9,25 @@ Ruby와 관련한 보안 이슈에 대해 정보를 공유하는 곳입니다.
...
+비공개 메일링 리스트인 security@ruby-lang.org ([the PGP public key](/security.asc))로 메일을 보내주십시오.
+
+이 메일링 리스트의 멤버는 Ruby를 제공하는 사람들(Ruby 커미터, Ruby 구현부의
 저자, 배포 담당자, PaaS 플랫폼 관리자)입니다.
-맴버는 반드시 개인이어야 하며, 메일링 리스트는 허용되지 않습니다.
+멤버는 반드시 개인이어야 하며, 메일링 리스트는 허용되지 않습니다.
 
 ## 알려진 취약점
 
-_See the [English page](/en/security/) for a complete and up-to-date
```
Comment (riseshia) [inline ko/security/index.md:22]: which can be covered by L31, so lets remove this.

File: ko/community/podcasts/index.md [hunk 2]
```diff
@@ -14,12 +14,17 @@ Ruby와 Ruby 커뮤니티에 대한 뉴스, 인터뷰, 토론을 들어보세요
 : Ruby on Rails 팟캐스트는 Ruby on Rails, 오픈 소스 소프트웨어, 프로그래밍
   업계에 대해 대화하는 주간 팟캐스트입니다.
 
+[Remote Ruby][remote_ruby]
+: 온라인 모임에서 팟캐스트가 된 Remote Ruby는 캐쥬얼한 분위기에서 Ruby 커뮤니티의
```
Comment (yous) [inline ko/community/podcasts/index.md:18]: How about this?

```suggestion
: 온라인 모임에서 팟캐스트가 된 Remote Ruby는 편안한 분위기에서 Ruby 커뮤니티의
```

File: ko/documentation/success-stories/index.md [hunk 3]
```diff
@@ -44,10 +44,10 @@ lang: ko
 
 #### 웹 애플리케이션
 
-* [Basecamp][9]는 [37signals][10]에서 개발한 웹기반 프로젝트 관리 툴입니다.
+* [Basecamp][9]는 개발한 웹기반 프로젝트 관리 툴입니다.
```
Comment (yous) [inline ko/documentation/success-stories/index.md:47]:
```suggestion
* [Basecamp][9]는 웹기반 프로젝트 관리 툴입니다.
```

File: ko/security/index.md [hunk 4]
```diff
@@ -9,21 +9,25 @@ Ruby와 관련한 보안 이슈에 대해 정보를 공유하는 곳입니다.
...
 
-이 메일링 리스트의 맴버는 Ruby를 제공하는 사람들(Ruby 커미터, Ruby 구현부의
+웹 사이트에 영향을 주는 문제를 발견한 경우에는 [GitHub](https://github.com/ruby/www.ruby-lang.org/issues/new)을
+통해서 보고하거나 [Google Group](https://groups.google.com/g/ruby-security-ann)의
+보안 공지를 확인하세요.
+
+Ruby 커뮤니티의 특정 gem에 영향을 주는 문제를 발견했다면, [RubyGems.org의 안내](http://guides.rubygems.org/security/#reporting-security-vulnerabilities)를 따라주세요.
+
+HackerOne 이외의 방법으로 보안팀에 연락하고 싶다면,
+비공개 메일링 리스트인 security@ruby-lang.org ([the PGP public key](/security.asc))로 메일을 보내주십시오.
```
Comment (yous) [inline ko/security/index.md:23]:
```suggestion
비공개 메일링 리스트인 security@ruby-lang.org([PGP public key](/security.asc))로 메일을 보내주십시오.
```

File: ko/security/index.md (hunk 1 above)
Comment (yous) [reply ko/security/index.md:22]: Let's update L31 bit.

``` markdown
아직 번역되지 않은 최근 취약점 및 자세한 사항은 [영문 페이지](/en/security/)를 참조하시기 바랍니다.
```

Comment (riseshia) [reply ko/security/index.md:22]: 👍  1ae6063a

**Review by yous — APPROVED** [review]

**Comment by riseshia** [comment]
Thanks!! :bow:

**Pull request merged**